### PR TITLE
feat(web): add contact management tab eligibility

### PR DESCRIPTION
**Pull request by antonymarion**
ref: MANAGER-15829

| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Feat#MANAGER-15829
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
~~- [ ] Breaking change is mentioned in relevant commits~~

### CODE REVIEW

File: packages/manager/apps/web/client/app/domain/contact/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "domain_tab_CONTACT_title": "Gérer les contacts propriétaires",
+  "domain_tab_CONTACT_modify": "Modifier",
+  "domain_tab_CONTACT_owner": "Titulaire",
+  "domain_tab_CONTACT_admin": "Administrateur",
+  "domain_tab_CONTACT_technical": "Technique",
+  "domain_tab_CONTACT_billing": "Billing",
+  "domain_tab_CONTACT_header": "Chaque nom de domaine est lié à 4 types de contacts. Chacun est associé à un identifiant client et dispose d'autorisations spécifiques.",
+  "domain_tab_CONTACT_description": "Seuls les rôles \"Propriétaire\" et \"Administrateur\" peuvent modifier ici les contacts associés au nom de domaine concerné. Toute mise à jour est immédiate.",
```
Comment (tristanwagner) [inline packages/manager/apps/web/client/app/domain/contact/translations/Messages_fr_FR.json:9]: "Propriétaire" => "Titulaire" ?

File: packages/manager/apps/web/client/app/domain/contact/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "domain_tab_CONTACT_title": "Gérer les contacts propriétaires",
+  "domain_tab_CONTACT_modify": "Modifier",
+  "domain_tab_CONTACT_owner": "Titulaire",
+  "domain_tab_CONTACT_admin": "Administrateur",
+  "domain_tab_CONTACT_technical": "Technique",
+  "domain_tab_CONTACT_billing": "Billing",
```
Comment (tristanwagner) [inline packages/manager/apps/web/client/app/domain/contact/translations/Messages_fr_FR.json:7]: No french translation for this one ?

Comment (antonymarion) [reply packages/manager/apps/web/client/app/domain/contact/translations/Messages_fr_FR.json:7]: Maybe Facturation, I am gonna ask thks

File: packages/manager/apps/web/client/app/domain/contact/translations/Messages_fr_FR.json (hunk 1 above)
Comment (antonymarion) [reply packages/manager/apps/web/client/app/domain/contact/translations/Messages_fr_FR.json:9]: Good Catch, Titulaire is better indeed!

**Review by tristanwagner — APPROVED** [review]

**Review by tristanwagner — APPROVED** [review]

File: packages/manager/apps/web/client/app/domain/contact/dashboard/dashboard.controller.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+      {
+        name: INFO_PROPERTIES.NATIONAL_IDENTIFICATION_NUMBER,
+        values: [INFO_PROPERTIES.NATIONAL_IDENTIFICATION_NUMBER],
+      },
+      {
+        name: INFO_PROPERTIES.NICHANDLE,
+        values: [INFO_PROPERTIES.NICHANDLE],
+      },
+    ];
+  }
```
Comment (ghyenne) [inline packages/manager/apps/web/client/app/domain/contact/dashboard/dashboard.controller.js:51]: i think you can move in contants file ?

**Review by tristanwagner — DISMISSED** [review]

File: packages/manager/apps/web/client/app/domain/contact/dashboard/dashboard.html [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+            data-translate="domain_tab_CONTACT_description_error_message"
+        >
+        </span>
+        <small data-ng-bind-html="$ctrl.saveError"></small>
+    </oui-message>
+
+    <div data-ng-show="!$ctrl.loading">
+        <div class="row" data-ng-if="!$ctrl.zone">
+            <div class="col-md-12">
+                <div data-ovh-alert="{{alerts.main}}"></div>
```
Comment (ghyenne) [inline packages/manager/apps/web/client/app/domain/contact/dashboard/dashboard.html:19]: you are sure that alerts.main is defined

Comment (antonymarion) [reply packages/manager/apps/web/client/app/domain/contact/dashboard/dashboard.html:19]: nope, remove it indeed!

File: packages/manager/apps/web/client/app/domain/contact/dashboard/dashboard.controller.js (hunk 3 above)
Comment (antonymarion) [reply packages/manager/apps/web/client/app/domain/contact/dashboard/dashboard.controller.js:51]: Yes also

**Review by tristanwagner — DISMISSED** [review]

**Review by tristanwagner — APPROVED** [review]

File: packages/manager/apps/web/client/app/domain/contact/dashboard/dashboard.controller.js [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+        const errorMessage = error.data?.message || error.data;
+        this.Alerter.error(
+          [
+            this.$translate.instant(
+              'domain_tab_CONTACT_description_error_message',
+            ),
+            errorMessage ? `(${errorMessage})` : '',
+          ].join(' '),
+          'dashboardContact',
+        );
```
Comment (ghyenne) [inline packages/manager/apps/web/client/app/domain/contact/dashboard/dashboard.controller.js:40]:
```suggestion
    this.DomainService.getServiceInfo(this.domainName)
      .then((data) => {
        this.domainInfos = data;
        this.isOwner = this.user.email === this.domain.whoisOwner.email;
      })
      .catch((error) => {
        const errorMessage = error.data?.message || error.data;
        this.Alerter.error(
          [
            this.$translate.instant(
              'domain_tab_CONTACT_description_error_message',
            ),
            errorMessage ? `(${errorMessage})` : '',
          ].join(' '),
          'dashboardContact',
        )
        .finally(() => {
          this.loading = false;
        });
```

File: packages/manager/apps/web/client/app/domain/contact/dashboard/dashboard.html [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+                    data-variant="link"
+                    data-icon-right="oui-icon-arrow-right"
+                    class="mb-2"
+                >
+                    <span data-translate="domain_tab_CONTACT_modify"></span>
+                </oui-button>
+            </oui-tile>
+            <oui-tile
+                class="col-md-3 m-1 align-self-stretch"
+                heading="{{:: 'domain_tab_CONTACT_admin' | translate }}"
```
Comment (ghyenne) [inline packages/manager/apps/web/client/app/domain/contact/dashboard/dashboard.html:32]:
```suggestion
                data-heading="{{:: 'domain_tab_CONTACT_admin' | translate }}"
```

File: packages/manager/apps/web/client/app/domain/contact/dashboard/dashboard.html [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+    <div data-ng-if="!$ctrl.loading">
+        <div class="row">
+            <strong data-translate="domain_tab_CONTACT_header"></strong>
+            <p data-translate="domain_tab_CONTACT_description"></p>
+        </div>
+
+        <div class="row d-flex align-items-stretch">
+            <oui-tile
+                class="col-md-3 m-1 align-self-stretch"
+                heading="{{:: 'domain_tab_CONTACT_owner' | translate }}"
```
Comment (ghyenne) [inline packages/manager/apps/web/client/app/domain/contact/dashboard/dashboard.html:18]:
```suggestion
                data-heading="{{:: 'domain_tab_CONTACT_owner' | translate }}"
```

**Review by tristanwagner — APPROVED** [review]

**Review by tristanwagner — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

**Pull request merged**